File: Programmers/Javascript/level1/문자열다루기기본.js
function solution(s) {
    var asclist=[];
    var answer=false
    if(s.length==4 || s.length==6)
        {
            for(var i=0;i<s.length;i++)
                {
                    asclist=s.split("").map(x=>x.charCodeAt([0]))
                }
            for(var i=0;i<asclist.length;i++)
            {
            if(asclist[i]<=57 && asclist[i]>=48)
                {
                    answer=true;
                } else {
                    answer=false
                    break
                }

            }
        } else {
            return false
        }
        return answer
}

File: Programmers/Java11/src/com/programmers/문자열다루기기본.java
package com.programmers;

public class 문자열다루기기본 {
    class Solution {
        public boolean solution(String s) {
            boolean answer = true;

            if(s.length() != 4 && s.length() !=6){
                return false;
            }

            for(int i=0; i<s.length(); i++){
                if (!Character.isDigit(s.charAt(i))){
                    answer = false;
                    break;
                }
            }

            return answer;
        }
    }

}
